docs/design.rst

```diff
@@ -23,7 +28,33 @@ Kafka is used as the Schema Registry storage backend. The special Kafka topic ``
 Single Master Architecture
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 The Schema Registry is designed to work as a distributed service using single master architecture. In this configuration, at most one Schema Registry instance is master at any given moment (ignoring pathological 'zombie masters'). Only the master is capable of publishing writes to the underlying Kafka log, but all nodes are capable of directly serving read requests. Slave nodes serve registration requests indirectly by simply forwarding them to the current master, and returning the response supplied by the master.
+Prior to Schema Registry version 4.0, master election was always coordinated through Zookeeper.
+Master election can now optionally happen via Kafka Coordinator protocol as well.
```

Comment:
coordinator is a loaded term for kafka. i'd refer to it as the Kafka group protocol, as that is less ambiguous (the kafka coordinator is also the broker that is managing the kafka cluster) and general (doesn't assume "consumer" although we normally refer to it as the "consumer group" protocol since we generalized it for connect, and now use it in SR).